spec/javascripts/components/story/collapsed_story/collapsed_story_actions_spec.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import CollapsedStoryPoints from 'components/story/CollapsedStory/CollapsedStoryPoints';
+import CollapsedStoryStateActions from 'components/story/CollapsedStory/CollapsedStoryStateActions';
+import storyFactory from '../../../support/factories/storyFactory';
+
+describe('<CollapsedStoryStateActions />', () => {
+  describe('When estimate is null', () => {
+    it("renders <CollapsedStoryPoints /> component", () => {
+      const props = storyFactory({estimate: null});
+      const wrapper = shallow(<CollapsedStoryStateActions {...props} />);
+      expect(wrapper.find(CollapsedStoryPoints)).toExist();
+    });
+  });
```

Comment:
Can we add a blank line between describes like this? It'd make easier to read 😄

Reply:
Yes, we can